jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -925,4 +925,11 @@ public int classFileVersion(Class<?> clazz) {
 		return clazz.getClassFileVersion();
 	}
 /*[ENDIF] (JAVA_SPEC_VERSION >= 25) & !INLINE-TYPES */
+
+/*[IF JAVA_SPEC_VERSION >= 26]*/
```

Comment:
Could you add `!INLINE-TYPES` like L922 to avoid Valhalla build breakage since it hasn't have latest openjdk content?